app/pages/project/home.cjsx

```diff
@@ -49,6 +51,27 @@ module.exports = React.createClass
             <strong>Visit the project</strong><br />
             <small>at {@props.project.redirect}</small>
           </a>
+        # For Gravity Spy
+        else if @props.project.experimental_tools.indexOf 'workflow assignment' isnt -1
```

Comment:
I think `indexOf 'workflow assignment'` should be `indexOf('workflow assignment')` otherwise I get stuck in the Gravity Spy condition?